flycheck-hledger.el

```diff
@@ -77,39 +77,39 @@ https://hledger.org/hledger.html#check."
 
    ;; hledger 1.26 assertions (:LINE:COL)
    (error
-    bol "hledger: Error: balance assertion: " (file-name (minimal-match (one-or-more (not ":")))) ":" line ":" column "\n"
-    (message (one-or-more bol (zero-or-more nonl) "\n")))
+    bol "hledger" (optional ".exe") ": Error: balance assertion: " (file-name) ":" line ":" column line-end
```

Comment:
it would make sense that the commit introducing this fix also introduces the test for this fix. This way the commit is atomic and can be merged (and potentially later reverted) separately. An atomic commit is also easier to understand because it contains everything that makes sense together and nothing more.